notation/install.ts

```diff
@@ -0,0 +1,46 @@
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+
+import { getDownloadInfo, installFromURL } from './lib/install';
+
+const NOTATION_VERSION_FILE = 'notation_versions.json';
+
+export async function install(): Promise<void> {
+    // the notation download URL
+    let version: string
+    let downloadURL: string
+    let checksum: string
+
+    if (taskLib.getBoolInput('isCustomVersion', false)) {
+        // for custom version, download the notation from specified URL
+        version = "1.0.0-UNRELEASED"
+        downloadURL = taskLib.getInput('url', true) || '';
```

Comment:
> function getInput(name: string, required?: boolean | undefined): string | undefined

Because the return value type is `string | undefined`, I want to simplify the `downloadURL` type to be `string` instead of `string | undefined`